patient-appointment-management/.env

```diff
@@ -0,0 +1,88 @@
+# description: Customer name to be displayed in SMS.
+# format: text
+# required: true
+# configurable: true
+CUSTOMER_NAME=
+
+# description: Customer short name. Must be all lowercase one word without any spaces.
```

Comment:
Is it possible to convert this to lowercase and remove spaces when this is used by the Function?

Reply:
Do you mean 'CUSTOMER_NAME' to 'customername'? Vaccine stand-by list app had environment variables in all upper-snake-case, so I thought that was preferred...

Reply:
Oh, you mean have our code convert to lowercase & remove spaces. I can do that.